pkg/secrets/check_rights_windows_test.go

```diff
@@ -50,7 +51,8 @@ func TestCheckRights(t *testing.T) {
 		"-removeAdmin", "0",
 		"-removeLocalSystem", "0",
 		"-addDDuser", "0").Run()
-	require.NotNil(t, checkRights(tmpfile.Name(), allowGroupExec))
+	err = checkRights(tmpfile.Name(), allowGroupExec)
+	assert.NotNil(t, checkRights(tmpfile.Name(), allowGroupExec))
```

Comment:
Given that these are chained tests, I could be wrong but I think the `require` is more desired than `assert` so that it can immediately exit.

Reply:
This may be a matter of opinion, but I actually think it's better the other way around so if something is wrong the suite continues to run and we can see all failures at completion; that's why I changed it.

Reply:
Ah, you're completely right - they're independent tests rather than a single test with chained logic. I guess this could probably be a set of parameterized tests or independent tests but it's fine as-is so I'll resolve.